docs/rules/wrap-iife.md

```diff
@@ -18,12 +18,18 @@ This rule requires all immediately-invoked function expressions to be wrapped in
 
 ## Options
 
-The rule takes one option which can enforce a consistent wrapping style:
+This rule has two options, a string option and an object option.
```

Comment:
Just prefer it to be grammatically correct - we can update the wording in the other rule to whatever we decide on here :)